fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java

```diff
@@ -1662,6 +1666,78 @@ public void updatePartitionTabletMeta(Database db,
         }
     }
 
+    public void updateTableConstraint(Database db, String tableName, Map<String, String> properties)
+            throws DdlException {
+        if (!db.readLockAndCheckExist()) {
+            throw new DdlException(String.format("db:%s does not exists.", db.getFullName()));
+        }
+        TableProperty tableProperty;
+        OlapTable olapTable;
+        try {
+            Table table = db.getTable(tableName);
+            if (table == null) {
+                throw new DdlException(String.format("table:%s does not exist", tableName));
+            }
+            olapTable = (OlapTable) table;
+            tableProperty = olapTable.getTableProperty();
+        } finally {
+            db.readUnlock();
+        }
+
+        boolean hasChanged = false;
+        if (tableProperty != null) {
+            if (properties.containsKey(PropertyAnalyzer.PROPERTIES_UNIQUE_CONSTRAINT)) {
```

Comment:
I see other systems cannot change constraints (eg, mysql: https://dev.mysql.com/doc/refman/8.0/en/alter-table.html). Maybe we just support add/drop constraints?

Reply:
now we do not support full constraint semantics. we use properties to manage table constraint. so we should support modify constraints just like add/drop constraints in mysql.